adhocteam/Head-Start-TTADP
# [TTAHUB-147]: Update styles for menus on AR tables

## Description of change

Changes to visual styles related to the "X selected" capsule widget and filter menus. Also some a11y cleanup.


## How to test

Start with multiple approved reports. On the "Activity reports" table, select one or more reports using the checkboxes, and a blue capsule widget should appear. Check that the capsule aligns with the other menus, and that all menus are ~16px font-size, per the designs. All text in the menu areas should be sans-serif, whereas before the capsule widget was rendering with a serif font.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-147


## Checklists

### Every PR


~Code is meaningfully tested~
~API Documentation updated~
~Boundary diagram updated~
~Logical Data Model updated~
~[Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
